gnovm/cmd/gno/test_test.go

```diff
@@ -186,5 +186,7 @@ func TestTest(t *testing.T) {
 		// {args: []string{"test", "../../tests/integ/failing2", "--verbose"}, stderrShouldBe: "=== PREC  ./../../tests/integ/failing2\n=== BUILD ./../../tests/integ/failing2\n=== RUN   file/failing_filetest.gno\n", recoverShouldBe: "fail on ../../tests/integ/failing2/failing_filetest.gno: got unexpected error: beep boop"},
 		// {args: []string{"test", "../../../examples/gno.land/p/demo/ufmt", "--verbose", "--timeout", "10000" /* 10µs */}, recoverShouldContain: "test timed out after"}, // FIXME: should be testable
 	}
-	testMainCaseRun(t, tc)
+	for i := 0; i < 100; i++ {
+		testMainCaseRun(t, tc)
+	}
```

Comment:
I imagine you added this to provide the pprof visualizations you shared with us. Do you want to keep this?

Maybe have this as a different test which is skipped with `t.Short()`?

Reply:
Oops, yes shouldn't have been committed. Will revert.

Reply:
```suggestion
	testMainCaseRun(t, tc)
```